### DISCUSSION TITLE
v2.1.0-A

### DISCUSSION THREAD

**Discussion by gianni-rosato** [gianni-rosato/svt-av1-psy#57] [Announcements]
Hi there! We're back with our micro-release format to announce some exciting changes in **SVT-AV1-PSY v2.1.0-A**! 🎉

### Features
- New parameter `--max-32-tx-size`, which restricts block transform sizes to a maximum of 32x32 pixels. This can be useful in very specific scenarios for improving overall efficiency
- Added support for HDR10+ JSON files via a new `--hdr10plus-json` parameter (thanks @quietvoid!). In order to build a binary with support for HDR10+, see our [PSY Development page](https://github.com/gianni-rosato/svt-av1-psy/blob/master/Docs/PSY-Development.md).
- New parameter `--adaptive-film-grain`, which helps remedy perceptually harmful grain patterns caused by extracting grain from blocks that are too large for a given video resolution. This parameter is enabled by default

### Quality & Performance
- Disabled SSIM-cost transform decisions while keeping SSIM-cost mode decisions, generally improving Tune 3's efficiency & consistency
- Additional NEON optimizations for ARM platforms, providing a speed increase

### Documentation
- All of the features present in this release have been documented, so associated documentation has been updated accordingly
- Build documentation updated to reflect the HDR10+ build option

### Bug Fixes
- Disabled quantization matrices for presets 5 and higher due to a visual consistency bug (#56)

*Thanks for using SVT-AV1-PSY!* ♥️

**Full Changelog**: https://github.com/gianni-rosato/svt-av1-psy/commits/v2.1.0-A

Linux builds: *(coming soon)*

macOS builds: *(coming soon)*

Windows builds: *(coming soon)*

<hr /><em>This discussion was created from the release <a href='https://github.com/gianni-rosato/svt-av1-psy/releases/tag/v2.1.0-A'>v2.1.0-A</a>.</em>

**Comment by gitoss** [comment]
Thanks - looking forward to the '(coming soon)' builds including Windows.

**Reply by gianni-rosato** [reply]
Hi, a Windows build for AVX2-capable CPUs is now available. Check the release binaries. Thanks for your patience!

**Reply by gitoss** [reply]
> Hi, a Windows build for AVX2-capable CPUs is now available.

Thanks - one question though: Does 'for AVX2-capable CPUs' mean that AVX512 is disabled? Using Zen 4 and --asm 512, only AVX2 is selected. I don't know if this is due to SVT itsself or this specific binary. 

```
SVT [version]: SVT-AV1-PSY Encoder Lib v2.1.0-A
Svt[info]: [asm level on system : up to avx512]
Svt[info]: [asm level selected : up to avx2]

```

Searching the web, it had to be enabled for Handbrake. Enabling AVX512 shouldn't hurt because SVT has asm auto-selection... https://github.com/HandBrake/HandBrake/issues/5596

**Reply by gitoss** [reply]
With the help of Patman86, I've figured out how to build svt with AVX512 - it needs manually added -DENABLE_AVX512=ON in cmake. https://github.com/Patman86/SVT-AV1-Mod-by-Patman/issues/6
